### DISCUSSION TITLE
Slow loading

### DISCUSSION THREAD

**Discussion by omkarkhallar** [vercel/next.js#53242] [App Router] [answered]
I am using Nextjs for 1month but I maked a project in Nextjs in that project the Link tag is very very very slow it take upto 2 minutes to go to next page or on click back button the url changes but the page don't change I have to reload the page it is very slow to load new page iam using the latest version why is this happening is there any solution?

**Comment by dhedegaard-digisense** [comment]
If you reload the browser on the destination page, do you then experience the same slow response times?

**Reply by omkarkhallar** [reply]
Yes

**Reply by omkarkhallar** [reply]
It is slow as on another page

**Reply by dhedegaard-digisense** [reply]
I have experienced the samme in my apps if the SSR rendering of the page is slow.

I would suggest that you try and profile your code for the pages that appear slow on a clean load in the browser.

**Comment by timneutkens** [comment]
Try upgrading Next.js first, most of these reports are not using the latest version.

**Comment by omkarkhallar** [comment]
![IMG_20230728_183255](https://github.com/vercel/next.js/assets/84460220/8aa0fa7f-a5bb-4dd3-af93-2a262d8bebef)

**Reply by omkarkhallar** [reply]
May be it's latest verson

**Reply by omkarkhallar** [reply]
?

**Reply by timneutkens** [reply, marked as answer]
That is definitely not the latest version 😅 Please upgrade using `npm install next@latest`.

**Reply by omkarkhallar** [reply]
Yes sir it worked but 10-20 sec delay is there

**Comment by omkarkhallar** [comment]
https://github.com/vercel/next.js/assets/84460220/594c7d5b-1b31-4b0a-b171-9e8d0d55a7da

**Comment by omkarkhallar** [comment]
There the delay is as it is ...

**Comment by omkarkhallar** [comment]
Upgrade the next version
